groups/sig-release/groups.yaml

```diff
@@ -191,6 +191,16 @@ groups:
     members:
       - k8s-infra-release-editors@kubernetes.io
 
+  - email-id: k8s-infra-staging-releng-test@kubernetes.io
+    name: k8s-infra-staging-releng-test
+    description: |-
+      ACL for sig k8s-releng-test artifacts
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - sig-release-leads@kubernetes.io
```

Comment:
```suggestion
```

We should only use the `k8s-infra-release-*` groups to delegate access.
`k8s-infra-release-editors@kubernetes.io` is sufficient here.